gimme

```diff
@@ -256,6 +277,28 @@ _try_git() {
 	echo "export GIMME_ENV=\"$(_env_alias "${git_dir}" "${git_env}")\""
 }
 
+_wipe_version() {
+	local env_file="${GIMME_ENV_PREFIX}/go${1}.${GIMME_OS}.${GIMME_ARCH}.env"
+
+	if [[ -s "${env_file}" ]] ; then
+		rm -rf "$(awk -F\" '/GOROOT/ { print $2 }' "${env_file}")"
+		rm -f "${env_file}"
+	fi
+}
+
+_realpath() {
+	[[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
```

Comment:
So, this feels a little bit hacky, and I think something like the following will be somewhat more robust:
```console
[ -d "$1" ] && echo "$(cd "$1" && pwd)" || echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
```

ie:
```console
$ cat temp.sh
#!/bin/bash
set -e

_realpath() {
	echo -n "$1 -->> "
	[ -d "$1" ] && echo "$(cd "$1" && pwd)" || echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
}

_realpath .
_realpath ..
_realpath /tmp/..
_realpath /home/../home/tianon/
_realpath "$BASH_SOURCE"
$ bash temp.sh
. -->> /home/tianon
.. -->> /home
/tmp/.. -->> /
/home/../home/tianon/ -->> /home/tianon
temp.sh -->> /home/tianon/temp.sh
```